doc/tutorial_sector.rst

```diff
@@ -0,0 +1,637 @@
+.. SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _tutorial_sector:
+
+##########################################
+Tutorial: Sector-Coupled
+##########################################
+
+.. note::
+    If you have not yet installed PyPSA-Earth, please refer to the :ref:`installation` section.
+
+In this tutorial, we will show you how to run the sector-coupled model. The sector-coupled model
+is a model that considers the energy system as a whole, including the electricity, heat, transport,
+and industry sectors. The model is based on the PyPSA-Eur model, which is a model of the European
+energy system. The sector-coupled model is a global model that can be used to model any region
+of the Earth. This section explains how to run and analyze the tutorial model.
+
+
+The sector-coupling code can be run as an overnight / greenfield scenario or myopic scenario.
+The overnight scenario is a long-term scenario that runs for a year, while the myopic scenario
+is a short-term scenario that runs for a day.
+
+
+Overnight Scenarios
+=============================================
+
+Configuration
+---------------------------------------------
+
+The configuration for sector-coupled is present in the  ``config.default.yaml`` file.
+This foresight parameter is set to overnight by default.
+
+.. code:: yaml
+
+    foresight: overnight
+
+Documentation for all options will be added successively to :ref:`config`.
+
+Scenarios can be defined like for electricity-only studies, but with additional wildcard options.
```

Comment:
May it be also a good idea to mention the retrieval flags, `retrieve_databundle` and `retrieve_databundle_sector`?  Having them on if not intended can be quite detrimental, unfortunately [saying from the experience]. So, I'd suggest to make sure that we have this part highlighted in the documentation as clear as possible

Reply:
Perfect, thank you!